src/OpenDB/test/gcd_abstract_lef.lefok

```diff
@@ -0,0 +1,812 @@
+      VIA 24.14 92.4 via2_960x340 ;
+      VIA 24.14 92.4 via1_960x340 ;
+      VIA 136.14 86.8 via3_960x340 ;
+      VIA 136.14 86.8 via2_960x340 ;
+      VIA 136.14 86.8 via1_960x340 ;
+      VIA 24.14 86.8 via3_960x340 ;
+      VIA 24.14 86.8 via2_960x340 ;
+      VIA 24.14 86.8 via1_960x340 ;
+      VIA 136.14 81.2 via3_960x340 ;
+      VIA 136.14 81.2 via2_960x340 ;
+      VIA 136.14 81.2 via1_960x340 ;
+      VIA 24.14 81.2 via3_960x340 ;
+      VIA 24.14 81.2 via2_960x340 ;
+      VIA 24.14 81.2 via1_960x340 ;
+      VIA 136.14 75.6 via3_960x340 ;
+      VIA 136.14 75.6 via2_960x340 ;
+      VIA 136.14 75.6 via1_960x340 ;
+      VIA 24.14 75.6 via3_960x340 ;
+      VIA 24.14 75.6 via2_960x340 ;
+      VIA 24.14 75.6 via1_960x340 ;
+      VIA 136.14 70 via3_960x340 ;
+      VIA 136.14 70 via2_960x340 ;
+      VIA 136.14 70 via1_960x340 ;
+      VIA 24.14 70 via3_960x340 ;
+      VIA 24.14 70 via2_960x340 ;
+      VIA 24.14 70 via1_960x340 ;
+      VIA 136.14 64.4 via3_960x340 ;
+      VIA 136.14 64.4 via2_960x340 ;
+      VIA 136.14 64.4 via1_960x340 ;
+      VIA 24.14 64.4 via3_960x340 ;
+      VIA 24.14 64.4 via2_960x340 ;
+      VIA 24.14 64.4 via1_960x340 ;
+      VIA 136.14 58.8 via3_960x340 ;
+      VIA 136.14 58.8 via2_960x340 ;
+      VIA 136.14 58.8 via1_960x340 ;
+      VIA 24.14 58.8 via3_960x340 ;
+      VIA 24.14 58.8 via2_960x340 ;
+      VIA 24.14 58.8 via1_960x340 ;
+      VIA 136.14 53.2 via3_960x340 ;
+      VIA 136.14 53.2 via2_960x340 ;
+      VIA 136.14 53.2 via1_960x340 ;
+      VIA 24.14 53.2 via3_960x340 ;
+      VIA 24.14 53.2 via2_960x340 ;
+      VIA 24.14 53.2 via1_960x340 ;
+      VIA 136.14 47.6 via3_960x340 ;
+      VIA 136.14 47.6 via2_960x340 ;
+      VIA 136.14 47.6 via1_960x340 ;
+      VIA 24.14 47.6 via3_960x340 ;
+      VIA 24.14 47.6 via2_960x340 ;
+      VIA 24.14 47.6 via1_960x340 ;
+      VIA 136.14 42 via3_960x340 ;
+      VIA 136.14 42 via2_960x340 ;
+      VIA 136.14 42 via1_960x340 ;
+      VIA 24.14 42 via3_960x340 ;
+      VIA 24.14 42 via2_960x340 ;
+      VIA 24.14 42 via1_960x340 ;
+      VIA 136.14 36.4 via3_960x340 ;
+      VIA 136.14 36.4 via2_960x340 ;
+      VIA 136.14 36.4 via1_960x340 ;
+      VIA 24.14 36.4 via3_960x340 ;
+      VIA 24.14 36.4 via2_960x340 ;
+      VIA 24.14 36.4 via1_960x340 ;
+      VIA 136.14 30.8 via3_960x340 ;
+      VIA 136.14 30.8 via2_960x340 ;
+      VIA 136.14 30.8 via1_960x340 ;
+      VIA 24.14 30.8 via3_960x340 ;
+      VIA 24.14 30.8 via2_960x340 ;
+      VIA 24.14 30.8 via1_960x340 ;
+      VIA 136.14 25.2 via3_960x340 ;
+      VIA 136.14 25.2 via2_960x340 ;
+      VIA 136.14 25.2 via1_960x340 ;
+      VIA 24.14 25.2 via3_960x340 ;
+      VIA 24.14 25.2 via2_960x340 ;
+      VIA 24.14 25.2 via1_960x340 ;
+      LAYER metal7 ;
+        RECT  20.14 104.83 180.5 107.63 ;
+        RECT  20.14 24.83 180.5 27.63 ;
+      LAYER metal4 ;
+        RECT  135.66 22.23 136.62 182.17 ;
+        RECT  23.66 22.23 24.62 182.17 ;
+      LAYER metal1 ;
```

Comment:
Hmmm technically this is correct, but usually LEF abstracts generally only expose power pins on the topmost layer because it's not useful to see the pins on blocked layers.

I don't think this is blocking for this PR, but it should probably become a GitHub issue to fix in another PR. I think a good solution would be to include an optional TCL argument for power pin layers, with the default to be the topmost layer which has shapes on it. And maybe issue a warning if no power pins would be exported.